server/config/routes.js

```diff
@@ -950,6 +950,8 @@ exports.configure = function configure(app) {
   app.get('/lots/:uuid', lots.details);
   app.put('/lots/:uuid', lots.update);
   app.get('/lots/:uuid/assignments/:depot_uuid', lots.assignments);
+  app.post('/lots/:uuid/merge', lots.merge);
+  app.get('/lot/dupes/:label?/:entry_date?/:expiration_date?/:initial_quantity?/:inventory_uuid?', lots.getDupes);
```

Comment:
Now that you are using `req.query`, this represents the query string component of the URL (`?x=y&w=z`).  That means you no longer need any of these extra parameters after `/lots/dupes`.  This are _parameters_ found in `req.params`, not _query string options_ found in `req.query`.